src/shared/components/plots/WaterfallPlot.tsx

```diff
@@ -420,6 +420,8 @@ export default class WaterfallPlot<D extends IBaseWaterfallPlotData> extends Rea
         const dataPoints = _.filter(this.waterfallPlotData, (d:any) => this.props.highlight!(d) );
 
         const searchLabels = _.cloneDeep(dataPoints);
+        // add marker field to search label
+        _.each(searchLabels, (o) => o.search_label = true);
```

Comment:
we've been using camelCase, so let's call it `searchLabel`?

Reply:
Ok, updated to camelCase.